scripts/release-notes.txt

```diff
@@ -1,5 +1,14 @@
 Release Notes of the ESBMC model checker
 
+*** Version 6.10
+* Lots of bugs cases relating unions have been fixed
+* Compound Literals are now supported
+*  _ExtInt struct fields now have proper padding and alignment
+* A bug where the size of 3-dimensional arrays was being not being computed correctly has been solved
+* A new intrinsic for stack allocation has been added: __ESBMC_alloca
+* The Solidity frontend now has support for: bool literal and while
+* New options have been added: file-output and cex-output
```

Comment:
please include the `--` to make it clear what exactly the options are

Reply:
Done